setup-prompts.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs')
+
+module.exports = [{
+  type: "input",
+  name: "clientData",
+  message: "Client data directory",
+  default: "C:/Program Files (x86)/World of Warcraft/Data",
+  validate: function(value) {
+    try {
+      if (fs.lstatSync(value).isDirectory()) {

```

Comment:
What were to happen if this directory were to be removed after configuring it? It would crash at runtime like it currently does I guess?